components/table/ProductTable.tsx

```diff
@@ -20,6 +23,12 @@ const ProductTable = () => {
 
   const productListLength = productsList.length;
 
+  const handleLoadMoreData = () => {
+    // mocked response
+    setProductsList([...productsList, ...moreMockedProductList]);
```

Comment:
When do we plan to attach backend data? Is [TECH-451](https://govstack-global.atlassian.net/browse/TECH-451) blocking it?

Reply:
[TECH-408](https://govstack-global.atlassian.net/browse/TECH-408) is blocking it.
I'm going to attach backend data while working on [TECH-318](https://govstack-global.atlassian.net/browse/TECH-318)